rm/rm-api/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMNodeSourceEvent.java

```diff
@@ -150,6 +154,10 @@ public String getSourceDescription() {
         return this.nodeSourceDescription;
     }
 
+    public String getAdditionalInformations() {
+        return additionalInformations;
```

Comment:
why not having additionalInformations as Map<String, String> and handle this on the client side (e.g. portal)?

Reply:
Ok